lib/src/services/configuration-service.dart

```diff
@@ -1,11 +1,13 @@
 part of 'tmdb-service.dart';
 
 abstract class ConfigurationService {
-  final String _configPath = "3/configuration";
+  static const String _configPath = "3/configuration";
 
   Configuration _configuration;
   String _apiKey;
 
+  List<Country> _countries;
```

Comment:
Any reason to cache the countries in the configuration service?
It will be lost on application restart, making it not that useful for consumers, I think we should leave it to the end user of library to cache the data from configuration service into a persistent storage so that it survives app restarts.